integration

```diff
@@ -857,6 +857,7 @@
   "Sian-Lee-SA/Home-Assistant-Switch-Manager",
   "signalkraft/mypyllant-component",
   "sillyfrog/Automate-Pulse-v2",
+  "sirkirby/unifi-network-rules",
```

Comment:
This is invalid JSON.

Reply:
Thank you for the review @ludeeus. I believe that I'm following the same convention as the other integrations. More detail would be appreciated. Thank you.

Reply:
You are right, I was just confused by a lot of other PRs earlier.